python/src/rest-api/rest_api.aac

```diff
@@ -0,0 +1,55 @@
+plugin:
+  name: REST API
+  package: rest_api
+  description: |
+    An AaC plugin that provides a RESTful API for managing and interacting with
+    the AaC Python package.
+  commands:
+    - name: rest-api
+      help_text: Start a RESTful interface for interacting with and managing AaC.
+      run_before:
+        - plugin: Check AaC
+          command: check
+      run_after:
+        - plugin: Generate
+          command: generate
```

Comment:
Why do you think `check` and `generate` need to be incorporated with this command?